compiler/runtime/OMRAOTRelocationRuntime.cpp

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#include "runtime/OMRAOTRelocationRuntime.hpp"
+#include <string>
+
+TR::RelocationRuntime * OMR::OMRAOTRelocationRuntime::self()
+   {
+   return reinterpret_cast<TR::RelocationRuntime*>(this);
+   }
+
+void *
+OMR::OMRAOTRelocationRuntime::symbolAddress(char *symbolName)
+   {
+   std::string symbol(const_cast<const char *>(symbolName));
+   return _symbolLocation[symbol];
+   }
+
+void
+OMR::OMRAOTRelocationRuntime::registerLoadedSymbol(const char *&symbolName, void *&symbolAddress)
```

Comment:
Why *& ?

Reply:
Thank you for the comment. There is no apparent reason for that, it is copying artifact. Thank you for pointing that out.